exercices/part3/practice2.py

```diff
@@ -0,0 +1,6 @@
+
+
+if __name__ == "__main__":
+    """Create a program that gets a message from the user and then prints it out backwards."""
+    message = input("Put your message here and we will print it out backwards: ")
+    print("".join(reversed(message)))
```

Comment:
Another way of printing reverse is : 
print(message[::-1])

Reply:
ok, thank you for the hint, i will remember this for the future